README.md

```diff
@@ -1,32 +1,91 @@
 
-WP-CLI is a set of command-line tools for managing WordPress installations.
+Quick links: [Using](#using) | [Installing](#installing) | [Support](#support) | [Extending](#extending) | [Contributing](#contributing) | [Credits](#credits)
+
+## Using
+
+This project aims to provide command-line support for any action you can perform in the WordPress admin and many you can't. So you can [install a plugin](http://wp-cli.org/commands/plugin/install/):
+
+```
+$ wp plugin install rest-api
+```
+
+And [activate it](http://wp-cli.org/commands/plugin/activate/):
+
+```
+$ wp plugin activate rest-api
+```
+
+View the [Quick Start](http://wp-cli.org/docs/quick-start/) guide or jump to the [complete list of commands](http://wp-cli.org/commands/) for managing themes and plugins, importing and exporting data, performing database search-replace operations and more.
+
+## Installing
+
+The recommended way to install WP-CLI is to download the Phar build, mark it executable and place it in your PATH.
+
+Download the [wp-cli.phar](https://raw.github.com/wp-cli/builds/gh-pages/phar/wp-cli.phar) using `wget` or `curl`. For example:
+
+```
+$ curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+```
+
+Check if it is working:
+
+```
+$ php wp-cli.phar --info
+```
+
+To use it from the command line by typing `wp`, make the file executable and move it to somewhere in your PATH. For example:
+
+```
+$ chmod +x wp-cli.phar
+$ sudo mv wp-cli.phar /usr/local/bin/wp
+```
 
-Installing
-------------
-If you just want to use WP-CLI, see <http://wp-cli.org/#install>.
+If WP-CLI installed successfully, you should see something like this when you run `wp --info`:
 
-If you want to hack on WP-CLI, see [CONTRIBUTING.md](CONTRIBUTING.md).
+```
+$ wp --info
+PHP binary:    /usr/bin/php5
+PHP version:    5.5.9-1ubuntu4.14
+php.ini used:   /etc/php5/cli/php.ini
+WP-CLI root dir:        /home/wp-cli/.wp-cli
+WP-CLI packages dir:    /home/wp-cli/.wp-cli/packages/
+WP-CLI global config:   /home/wp-cli/.wp-cli/config.yml
+WP-CLI project config:
+WP-CLI version: 0.23.0
+```
 
-Where can I get more info?
---------------------------
-For documentation and examples, check out [wp-cli.org](http://wp-cli.org/) and the [documentation portal](http://wp-cli.org/docs/).
+Now that you've got WP-CLI, read the [Quick Start](http://wp-cli.org/docs/quick-start/) guide. Or view the [full installation guide](http://wp-cli.org/docs/installing/) with alternative installation methods.
 
-Also, WordPress Answers has a growing list of [WP-CLI related questions](http://wordpress.stackexchange.com/questions/tagged/wp-cli).
+## Support
 
-If you want to receive an email for every single commit, you can subscribe to the [wp-cli-commits](https://groups.google.com/forum/?fromgroups=#!forum/wp-cli-commits) mailing list.
+WP-CLI is a volunteer-led project and can't support every user individually. If you run into trouble, here are some places to look for help:
```

Comment:
I'm not quite sure about this language:

> WP-CLI is a volunteer-led project and can't support every user individually.

I do try to support everyone, but maybe we should have some communication of what types of issues are supported and what aren't.